docs/source/data_recipes/ERA5_preprocessing_example.md

```diff
@@ -100,56 +97,78 @@ variables according to the Virtual Rainforest naming convention (see
-dataset_cleaned = dataset.drop_vars(["d2m",'d2m_C',"t2m","stl1"])
+dataset_cleaned = dataset.drop_vars(["d2m",'d2m_C',"t2m"])
 dataset_renamed = dataset_cleaned.rename({
-    'si10':'wind_speed_ref',
     'sp':'atmospheric_pressure_ref',
     'tp':'precipitation',
-    'swvl1':'soil_moisture_ref',
     't2m_C':'air_temperature_ref',
     'rh2m': 'relative_humidity_ref',
-    'stl1_C': 'top_soil_temperature_ref',
-    'tisr':'TOA_incoming_solar_radiation',
-    'tcc':'total_could_cover',
     })
 dataset_renamed.data_vars
 ```
 
-### 5. Select required variables and add further required variables
+### 5. Add further required variables
 
-The set of required variables depends on the complexity of the abiotic model that is
-selected for the simulation. For the simple abiotic model, we require only a subset of
-the loaded climate data, and can therefore drop unrequired variables. In addition to the
-climatic variables, a time series of atmospheric CO2 is needed. We add this here as a
-constant field. Mean annual temperature is calculated from the full time series of air
-temperatures; in the future, this should be done for each year.
+In addition to the variables from the ERA5-Land datasset, a time series of atmospheric
+$\ce{CO_{2}}$ is needed. We add this here as a constant field. Mean annual temperature
+is calculated from the full time series of air temperatures; in the future, this should
+be done for each year.
 
 ```{code-cell} ipython3
-dataset_simple = dataset_renamed.drop_vars(
-  [
-    'wind_speed_ref',
-    'soil_moisture_ref',
-    'top_soil_temperature_ref',
-    'TOA_incoming_solar_radiation',
-    'total_could_cover',
-  ]
-)
-dataset_simple['atmospheric_co2_ref'] = DataArray(
-  np.full_like(dataset_simple['air_temperature_ref'],400),
+dataset_renamed['atmospheric_co2_ref'] = DataArray(
+  np.full_like(dataset_renamed['air_temperature_ref'],400),
   dims=['time', 'latitude', 'longitude'],
 )
-dataset_simple['mean_annual_temperature'] = (
-  dataset_simple["air_temperature_ref"].mean(dim = 'time')
+dataset_renamed['mean_annual_temperature'] = (
+  dataset_renamed["air_temperature_ref"].mean(dim = 'time')
   )
-dataset_simple
+dataset_renamed.data_vars
+```
+
+### 7. Change coordinates to x-y in meters
+
+The following code segment changes the coordinate names from `longitude/latitude` to
+`x/y` and the units from `minutes` to `meters`. The ERA5-Land coordinates are treated as
+the centre points of the grid cells which means that when setting up the grid, an offset
+of 4.5 km has to be added.
+
+```{code-cell} ipython3
+dataset_xy = dataset_renamed.rename_dims({'longitude':'x','latitude':'y'}).assign_coords({'x':np.arange(0,180000,9000),'y':np.arange(0,180000,9000)}).drop({'longitude','latitude'})
+dataset_xy.coords
+```
+
+### 8. Scale to 90 m resolution
+
+The Virtual Rainforest is run on a 90 x 90 m grid. This means that some form of spatial
+scaling has to be applied to the dataset, **which is not yet implemented!**
```

Comment:
Not strictly true - these data will run it at that resolution!

Reply:
I rephrased it to make this clearer